packages/ui/src/sidebar.tsx

```diff
@@ -0,0 +1,129 @@
+import type { FC } from 'react';
+import {
+	Dashboard,
+	LibraryBooks,
+	PermMedia,
+	Settings,
+	Storage,
+} from '@mui/icons-material';
+import {
+	Box,
+	Divider,
+	List,
+	ListItem,
+	ListItemButton,
+	Typography,
+} from '@mui/material';
+import Link from 'next/link';
+
+interface SidebarProps {
+	handleDrawerToggle: () => void;
+}
+
+interface SidebarItemProps {
+	href: string;
+	text: string;
+	icon: JSX.Element;
+}
+
+const SidebarItem: FC<SidebarItemProps> = ({ href, text, icon }) => (
+	<ListItem disablePadding sx={styles.listItem}>
+		<ListItemButton
+			component={Link}
+			href={href}
+			style={{ textDecoration: 'none' }}
+		>
+			{icon}
+			<Typography variant="body1" sx={styles.text}>
+				{text}
+			</Typography>
+		</ListItemButton>
+	</ListItem>
+);
+
+const Sidebar: FC<SidebarProps> = ({ handleDrawerToggle }) => {
```

Comment:
avoid having 2 more more components on the same file, it always better to split/separate them out.